website/docs/r/sagemaker_model.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "aws"
+page_title: "AWS: sagemaker_model"
+sidebar_current: "docs-aws-resource-sagemaker-model"
+description: |-
+  Provides a Sagemaker model resource.
+---
+
+# aws\_sagemaker\_model
+
+Provides a Sagemaker model resource.
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "aws_sagemaker_model" "m" {
+    name = "my-model"
+
+    primary_container {
+        image = "111111111111.ecr.us-west-2.amazonaws.com/my-docker-image:latest"
+    }
+}
+```
+
+Usage with supplemental container and tags:
+
+```hcl
+resource "aws_sagemaker_model" "m" {
+    name = "my-model"
+
+    primary_container {
+        image = "111111111111.ecr.us-west-2.amazonaws.com/my-docker-image:latest"
+        model_data_url  = "s3://111111111111-foo/model.tar.gz"
+    }
+
+    supplemental_container {
```

Comment:
Can this be removed? It doesn't exist in the provider code or the documentation on the aws side

Reply:
Done.